include/LightGBM/config.h

```diff
@@ -673,6 +673,7 @@ struct Config {
   // desc = **Note**: works only in case of loading data directly from file
   // desc = **Note**: data should be grouped by query\_id
   // desc = **Note**: index starts from ``0`` and it doesn't count the label column when passing type is ``int``, e.g. when label is column\_0 and query\_id is column\_1, the correct parameter is ``query=0``
+  // desc = for more information, see `Query Data <#query-data>`__
```

Comment:
Initially I thought to suggest reading explanatory `Query Data` section right after general words `should be grouped`. And that section explains how data should be grouped. WDYT? 
```suggestion
  // desc = **Note**: data should be grouped by query\_id, for more information, see `Query Data <#query-data>`__
  // desc = **Note**: index starts from ``0`` and it doesn't count the label column when passing type is ``int``, e.g. when label is column\_0 and query\_id is column\_1, the correct parameter is ``query=0``
```